config/initializers/rescue_from.rb

```diff
@@ -1,46 +1,47 @@
 require 'openstax_rescue_from'
 
 OpenStax::RescueFrom.configure do |config|
-  config.raise_exceptions = Rails.application.config.consider_all_requests_local
+  # Show the default Rails exception debugging page on dev
+  config.raise_exceptions = ENV['RAISE'].nil? ?
```

Comment:
I might be misunderstanding the intention of this; wouldn't checking for the rails dev env work instead?

Reply:
In dev and and test I guess we normally want the exceptions to be raised (dev displays the Rails exception rescue page with backtrace, test just tests that the thing was raised). I guess what this does is give us the option of passing `RAISE=false` so we can see the production rescue page in dev etc.